src/app/client/src/app/modules/shared/services/config/url.config.json

```diff
@@ -302,6 +302,9 @@
     },
     "RECAPTCHA": {
       "VALIDATE": "/validate/recaptcha"
+    },
+    "TENANT_PREFERENCE" : {
+      "READ": "org/v2/preferences/read"
```

Comment:
We have to onboard this API into dev-ops repo as well.

@swetha-aggidevara can you create the PR for dev-ops repo to onboard this API.

Reply:
this api is already been onboarded by UP team. isn't it ?

Reply:
No. We have to onboard on kong to allow the API request from portal

Reply:
Not requried